teuthology/report.py

```diff
@@ -579,6 +583,8 @@ def try_mark_run_dead(run_name):
             try:
                 log.info("Marking job {job_id} as dead".format(job_id=job_id))
                 reporter.report_job(run_name, job['job_id'], dead=True)
+                if "machine_type" in job:
```

Comment:
same as comment above

Reply:
it would be exceptionally strange (read: impossible?) for this to not be a real value, since paddles won't accept a job with no machine type